Doc/library/io.rst

```diff
@@ -1080,15 +1087,50 @@ Text I/O
       # .getvalue() will now raise an exception.
       output.close()
 
+      # docoding newlines for universal newlines mode.
+      io.StringIO('1\r\n2\r3\n4', newline=None).getvalue()
+      # '1\n2\n3\n4'
+
 
 .. index::
    single: universal newlines; io.IncrementalNewlineDecoder class
 
-.. class:: IncrementalNewlineDecoder
+.. class:: IncrementalNewlineDecoder(decoder, translate, errors='strict')
 
    A helper codec that decodes newlines for :term:`universal newlines` mode.
-   It inherits :class:`codecs.IncrementalDecoder`.
+   It inherits :class:`codecs.IncrementalDecoder`. It also records the types
+   of newlines encountered. When used with ``translate=False``, it ensures
+   that the newline sequence is returned in one piece. If an incremental
+   decoder is not required, :class:`TextIOWrapper` or :class:`StringIO`
+   should be used.
+
+   *decoder* is an incremental decoder or ``None``. If set to ``None`` it
+   decodes strings.
+
+   If *translate* is set to true, translate ``'\r\n'`` and ``'\r'`` into
+   ``'\n'``.
```

Comment:
What is the default value for translate?

Reply:
No default value.
https://github.com/python/cpython/blob/e18d81569fa0564f3bc7bcfd2fce26ec91ba0a6e/Lib/_pyio.py#L1905